waltz-tools/src/main/java/com/wepay/waltz/tools/storage/StorageCli.java

```diff
@@ -367,7 +366,7 @@ protected void configureOptions(Options options) {
                     .build();
             Option partitionOption = Option.builder("p")
                     .longOpt("partition")
-                    .desc("Specify the partition id to be added to the storage node")
+                    .desc("Specify the preferred partition (or multiple partitions as comma-separated int ranges such as 0-6,7,10-16) to be added to the given server")
```

Comment:
Instead of `added to the given server`, it should be `set as available to the given server`.